usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/setting/XWalkViewSettingSaveFormDataAsync.java

```diff
@@ -0,0 +1,101 @@
+package org.xwalk.embedded.api.asyncsample.setting;
+
+import org.xwalk.core.XWalkInitializer;
+import org.xwalk.core.XWalkSettings;
+import org.xwalk.core.XWalkView;
+import org.xwalk.embedded.api.asyncsample.R;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+public class XWalkViewSettingSaveFormDataAsync extends Activity implements XWalkInitializer.XWalkInitListener {
+
+    private XWalkInitializer mXWalkInitializer;
+
+    public final static String ENABLE_BT = "Set SaveFormData(ture)";
+    public final static String DISABLE_BT = "Set SaveFormData(false)";
+    public final static String MESSAGE_TITLE = "Get SaveFormData: ";
+
+    private XWalkView mXWalkView;
+    private TextView mMessage;
+    private Button mButton;
+    private XWalkSettings settings;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        mXWalkInitializer = new XWalkInitializer(this, this);
+        mXWalkInitializer.initAsync();
+    }
+
+    @Override
+    public void onXWalkInitCancelled() {
+        // TODO Auto-generated method stub
+    }
+
+    @Override
+    public void onXWalkInitFailed() {
+        // TODO Auto-generated method stub
+    }
+
+    @Override
+    public void onXWalkInitStarted() {
+        // TODO Auto-generated method stub
+    }
+
+    @Override
+    public void onXWalkInitCompleted() {
+        setContentView(R.layout.activity_xwalk_view_setting_save_form_data_async);
+
+        StringBuffer mess = new StringBuffer();
+        mess.append("Test Purpose: \n\n")
+                .append("Check if XWalkView can set/get SaveFormData in XWalkSettings.")
+                .append("Test Steps:\n\n")
+                .append("1. Set SaveFormData to true if it's false.\n\n")
+                .append("2. Input your name.\n\n")
+                .append("3. Click submit button.\n\n")
+                .append("4. Input first letter of your name or double click input area.\n\n")
+                .append("5. Set SaveFormData to false.\n\n")
+                .append("6. Input first letter of your name again or double click input area.\n\n")
+                .append("Expected Result:\n\n")
+                .append("Test passes if the saved form date does not auto complete(or popup) after set SaveFormData to false.");
+        new  AlertDialog.Builder(this)
+                .setTitle("Info")
+                .setMessage(mess.toString())
+                .setPositiveButton("confirm", null)
+                .show();
```

Comment:
Besides the popup, the test step should be shown in the main activity also as what we did in other use case